src/components/Link/Link.scss

```
@@ -29,11 +29,11 @@
 
   &:active {
     position: relative;
-    color: var(--p-interactive-pressed, color('blue', 'dark'));
+    color: var(--p-interactive-pressed);
     text-decoration: underline;
 
     &::before {
-      content: var(--p-non-null-content, none);
+      content: var(--p-non-null-content);
```

```suggestion
      content: '';
```

I think the token was just added as an override so we'll probably be removing it at some point